practices/mariorosales8/Practica2/Practica2.py

```diff
@@ -0,0 +1,171 @@
+import nltk
+from nltk.corpus import cess_esp
+import requests
+from sklearn.model_selection import train_test_split
+from nltk.tag import hmm
+from collections import defaultdict
+from inspect import Attribute
+from sklearn_crfsuite import CRF
+from sklearn.metrics import classification_report
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import precision_score
+from sklearn.metrics import recall_score
+from sklearn.metrics import f1_score
+
+
+def parse_list(str):
```

Comment:
No recomiendo usar como variable `str` porque es un método built-in de python
```suggestion
def parse_list(string):
```